src/components/Maker.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+const Maker = () => {
```

Comment:
Can we come up with better Name for this component.

Reply:
I changed it to UserCard for now. Also, Do we follow a certain convention for naming components?

Reply:
`CamelCase` working fine and mostly used for naming  Components.